src/test/java/de/tum/in/www1/artemis/notification/SingleUserNotificationServiceTest.java

```diff
@@ -1,6 +1,12 @@
 package de.tum.in.www1.artemis.notification;
 
-import static de.tum.in.www1.artemis.domain.enumeration.NotificationType.*;
+import static de.tum.in.www1.artemis.domain.enumeration.NotificationType.CONVERSATION_ADD_USER_CHANNEL;
```

Comment:
You can keep this star import. It's only important for external references. But we have to ship our own files anyway.